File: app/src/main/java/com/haishinkit/util/ILogAdapter.java
package com.haishinkit.util;

public interface ILogAdapter {
    public int v(String tag, String msg);
    public int v(String tag, String msg, Throwable tr);
    public int d(String tag, String msg);
    public int d(String tag, String msg, Throwable tr);
    public int i(String tag, String msg);
    public int i(String tag, String msg, Throwable tr);
    public int w(String tag, String msg);
    public int w(String tag, String msg, Throwable tr);
    public int w(String tag, Throwable tr);
    public int e(String tag, String msg);
    public int e(String tag, String msg, Throwable tr);
}

File: haishinkit/src/main/java/com/haishinkit/util/ILogAdapter.kt
package com.haishinkit.util

interface ILogAdapter {
    fun v(tag: String, msg: String): Int
    fun v(tag: String, msg: String, tr: Throwable): Int
    fun d(tag: String, msg: String): Int
    fun d(tag: String, msg: String, tr: Throwable): Int
    fun i(tag: String, msg: String): Int
    fun i(tag: String, msg: String, tr: Throwable): Int
    fun w(tag: String, msg: String): Int
    fun w(tag: String, msg: String, tr: Throwable): Int
    fun w(tag: String, tr: Throwable): Int
    fun e(tag: String, msg: String): Int
    fun e(tag: String, msg: String, tr: Throwable): Int
}
